### DISCUSSION TITLE
Bat doesn't recognize that its the manpage.

### DISCUSSION THREAD

**Discussion by azizsayfiddin** [sharkdp/bat#3027] [Q&A]
I set the `MANPAGER` environment variable to `bat -l man`, but it doesn't render the man page as it would if I ran `man bat | bat -l man`. How can I make it recognize the manpages as manpages

**Comment by keith-hall** [comment]
Piping a `man` invocation into something else changes the way `man` outputs the manpage - it removes formatting basically. i.e. `man bat | cat` is already different from `man bat`. Indeed, `bat` wasn't designed to syntax highlight text which already contains ANSI escape sequences (though a recent change was implemented to cater for this: https://github.com/sharkdp/bat/pull/2999) or weird use of backspace characters to repeat a character to make it bold typewriter style.
The guide for setting the `MANPAGER` env var (among others) to remove the odd formatting `man` applies and thus let `bat` handle it is in the `bat` readme: https://github.com/sharkdp/bat?tab=readme-ov-file#man
